### DISCUSSION TITLE
Removing unused sprinkle classes

### DISCUSSION THREAD

**Discussion by Floffah** [vanilla-extract-css/vanilla-extract#404] [Ideas]
After using this for the first time, I have a few issues, but there is only one that u find annoying.
As you know, in tailwind it will remove classes from the production build css file where they aren't used. From my experience it seems apparent that a new class for each possible sprinkle properties value is created even if unused. Since I am using sprinkles to sort of create a design system (not really) this bloats the production css file a lot where unnecessary. For now I will just uncomment unused values and remove it from my key typings however in the future I feel like a dynamic class remover solution would be great even if just available to frameworks like nextjs, etc via their respective plugin.

Enough negative, I love this library and will definitely be switching from tailwind + styled components to vanilla extract css and sprinkles!

**Comment by mattcompiles** [comment]
We've discussed this issue before. [You can see my previous reply here](https://github.com/seek-oss/vanilla-extract/discussions/91#discussioncomment-696695).
